lib/core/services/authentication.service.ts

```diff
@@ -116,7 +112,7 @@ export class AuthenticationService {
      * @returns Response event called when logout is complete
      */
     logout() {
-        this.removeTicket();
+        this.alfrescoApi.getInstance().invalidateSession();
```

Comment:
do we need invalidating before logout call or after?

Reply:
I would prefer to do it before even if the call fail anyway at least you know that on that machine your ticket is not present anymore